package.json

```diff
@@ -31,7 +31,8 @@
     "build-test": "rollup -c test/rollup.unit.config.js",
     "build-uglify": "rollup -c && uglifyjs build/three.js -cm --preamble \"// threejs.org/license\" > build/three.min.js",
     "build-closure": "rollup -c && java -jar node_modules/google-closure-compiler/compiler.jar --warning_level=VERBOSE --jscomp_off=globalThis --jscomp_off=checkTypes --externs utils/build/externs.js --language_in=ECMASCRIPT5_STRICT --js build/three.js --js_output_file build/three.min.js",
-    "dev": "rollup -c -w -m inline",
+    "dev": "concurrently --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c -w -m inline\" \"http-server\"",
+    "start": "npm run dev",
```

Comment:
Do you mind removing this line? 😇

Reply:
> @mrdoob `start` is the default in most of the projects, in fact with `start` you don't need to use `run` so you could just launch with `npm start`.
And so as soon as I clone a new project I use to do `npm install && npm start` otherwise I need to check on the package and it could be confusing as new commands will appear.

Bueno... Aceptamos barco 😜